central/globaldb/postgres.go

```diff
@@ -104,14 +104,6 @@ func InitializePostgres(ctx context.Context) postgres.DB {
 			log.Fatalf("Could not parse postgres config: %v", err)
 		}
 
-		if !pgconfig.IsExternalDatabase() {
```

Comment:
for ACS hosted Posgres, with this PR `central_active` will be created as part of the migrator.  So with the exception of restores, central will not need to do anything with the database name.